lib/pure/collections/tables.nim

```diff
@@ -1068,22 +1072,28 @@ proc largest*[A](t: CountTable[A]): tuple[key: A, val: int] =
   result.key = t.data[maxIdx].key
   result.val = t.data[maxIdx].val
 
-proc sort*[A](t: var CountTable[A]) =
+proc sort*[A](t: var CountTable[A], order: SortOrder = SortOrder.Descending) =
   ## sorts the count table so that the entry with the highest counter comes
-  ## first. This is destructive! You must not modify ``t`` afterwards!
-  ## You can use the iterators ``pairs``, ``keys``, and ``values`` to iterate over
-  ## ``t`` in the sorted order.
+  ## first. Set `order` to ``SortOrder.Ascending`` to reverse the order.
+  ## (Note that ``SortOrder`` is defined in ``algorithm``, and is re-exported by
+  ## ``tables`` for convenience.)
+  ##
+  ## This operation is destructive! You must not modify `t` afterwards! You can
+  ## use the iterators `pairs`, `keys`, and `values` to iterate over `t` in the
+  ## sorted order.
 
   # we use shellsort here; fast enough and simple
   var h = 1
+  # order_factor reverses the comparison on ascending sorts.
+  let order_factor = (if order == SortOrder.Descending: 1 else: -1)
```

Comment:
Convention is to use camelCase here :)

Reply:
all names changed to camelCase in 60d627f .